source/databricks/calculation_engine/package/settlement_report_job/table_column_names.py

```diff
@@ -0,0 +1,22 @@
+class TableColumnNames:
+    grid_area_code = "grid_area_code"
+    observation_time = "observation_time"
+    resolution = "resolution"
+    calculation_type = "calculation_type"
+    calculation_id = "calculation_id"
+    metering_point_id = "metering_point_id"
+    metering_point_type = "metering_point_type"
+    quantity = "quantity"
+
+
+class TimeSeriesPointColumnNames:
```

Comment:
Column names in view or CSV?
```suggestion
class TimeSeriesPointCsvColumnNames:
```